src/agents/constants.ts

```diff
@@ -0,0 +1,35 @@
+interface Contracts {
+  NU: string;
```

Comment:
Is it worth having a `ChecksumAddress` type? Even if it's just an alias for a `string`?

Reply:
Ah, looking in the next file, and you already have a `ChecksumAddress` defined. Could it be used here?